### PR TITLE
workaround stock sqlite token count limit

### PR DESCRIPTION
**Pull request by aakash10292**
exclude token count check as we've hit stock sqlite limit. Also expand Expr.op field to u16 since we now have more than 255 tokens

### CODE REVIEW

**Review by roborivers — COMMENTED** [review]
Lint check:

```diff


@@ sqlite/src/parse.y @@
+ Custom tokens and rules must be defined after SQLite's.
+ Please update SQL documentation and bubble diagrams to reflect the changes you've made, if necessary.
```

**Review by morgando — DISMISSED** [review]
No Wconversion errors from this change -- LGTM once the robots approve!

File: sqlite/src/sqliteInt.h
Comment (riverszhang89) [inline sqlite/src/sqliteInt.h:2597]: I'd ifdef-guard this as well.

Comment (aakash10292) [reply sqlite/src/sqliteInt.h:2597]: will do

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>8/532 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>sc_resume<br>truncatesc<br>schemalk_extralock_generated<br>truncatesc_offline_generated<br>comdb2sys_queueodh_generated<br>comdb2sys_pagesize_generated<br>comdb2sys</p>

**Comment by aakash10292** [comment]
cdb2test   May  2 22:45:23 2024  in-comdb2db aar_increasing_token_limit.R20240502.10

**Review by mponomar — APPROVED** [review]

**Pull request merged**
